index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"/>
+
+	<title>My Portifolio Page</title>
+	<link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
```

Comment:
Your design is not matching the Figma template of your choosing. Consider updating it accordingly. It is super important at this point that you follow the Figma design to its core, which includes, layout, spacing, margins, padding, colors, font, font size, etc. So kindly, update your design accordingly.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/185301014-32867985-bff2-4b79-901a-2be91e03de1c.png) | ![image](https://user-images.githubusercontent.com/33268927/185300932-9a39ce31-9ca3-46b3-859b-41a3d861fe46.png) |